eisbuk-admin/src/components/atoms/AttendanceSheet/AttendanceSheetSlot.tsx

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+
+import TableBody from "@material-ui/core/TableBody";
+import TableRow from "@material-ui/core/TableRow";
+import Table from "@material-ui/core/Table";
+import TableCell from "@material-ui/core/TableCell";
+import TableHead from "@material-ui/core/TableHead";
+
+import makeStyles from "@material-ui/core/styles/makeStyles";
+
+import { CustomerWithAttendance, SlotInterface } from "@/types/temp";
+
+import { getSlotTimespan } from "@/utils/helpers";
+
+/** @TODO remove this and import same type when it's merged*/
+export interface Props extends SlotInterface {
+  /**
+   * Record of customers who have booked (or are manually as attended),
+   * keyed by `customerId` and having all of the data for the customer plus
+   * values for `bookedInterval` and `attendedInterval`.
+   */
+  customers: CustomerWithAttendance[];
+}
+
+const AttendanceSheetSlot: React.FC<Props> = ({
+  customers,
+  intervals,
+  type,
+  notes,
+}) => {
+  const classes = useStyles();
+
+  const timeString = getSlotTimespan(intervals);
+
+  /** UI todos:
+   * @TODO in page component render alternative background shades for slot table
+   * @TODO center each slot timestring
+   * @TODO justify spaces in rows with 2 columns (name cell is much bigger than signature cell)
+   *
+   */
+  return (
+    <Table aria-label="simple table">
+      <TableHead className={classes.timeString}>
+        <TableRow>
```

Comment:
1. Instead of assigning a `timeString` class to `TableHead` and targeting `.MuiTableRow-root` nested underneath, you can assign a `timeString` class to `TableRow`, which is the element you would be targeting with `timeString .MuiTableRow-root`. This way it's more straightforward/specific. It slightly optimizes CSS performance, which really isn't an issue here, but rather being cleaner with the code.
2. Maybe name this `slotHeading` or just `heading` since it covers the entire heading element rather than just the time string